lib/utils.js

```
@@ -25,6 +25,9 @@ utils = {
       '/' + stores.publish.expand( utils.generateIdString( id ) ) );
   },
   generateIframeUrl: function( id ) {
+    if ( !id ) {
```

Why would we not have an `id`?  This looks like you're compensating for bugs in callers.  I get nervous when I see functions silently returning on error cases.